guides/session.rst

```diff
@@ -80,42 +120,78 @@ With ``Behat\Mink\Session`` you can control your browsers cookies and headers:
     // retrieving response headers:
     print_r($session->getResponseHeaders());
 
-    // set cookie:
-    $session->setCookie('cookie name', 'value');
-
-    // get cookie:
-    echo $session->getCookie('cookie name');
-
-    // delete cookie:
-    $session->setCookie('cookie name', null);
-
 .. note::
 
     Headers handling is only supported in headless drivers, because there
```

Comment:
Maybe something like:

... headless drivers (e.g. Goutte). Browser drivers (e.g. Selenium2) cannot access that information.